### DISCUSSION TITLE
Project Failing to Build on Visual Code Editor SDK Extension

### DISCUSSION THREAD

**Discussion by gtherezan** [migueesc123/PowerBIRESTAPI#173] [General]
Hi!

I'm trying to clone and build this project on my own as I plan to try to extend some of its functionality to include both WidelySharedArtifacts endpoints:
https://learn.microsoft.com/en-us/rest/api/power-bi/admin/widely-shared-artifacts-links-shared-to-whole-organization
https://learn.microsoft.com/en-us/rest/api/power-bi/admin/widely-shared-artifacts-published-to-web

The automatically build .mez file doesn't works. It even fails the default test on .query.pq.

Do you know if this project is compatible with the new recommended SDK on Visual Studio Code?

Thanks!

**Comment by migueesc123** [comment]
hey! This was created with the previous version of the SDK. There's a mismatch between how projects get created between the old and new SDK.
You might be missing to setup your workspace for this new project.

**Reply by gtherezan** [reply]
I've read the docs and the default process to "convert" the project to the new SDK format doesn't work.

So it won't work =/

I will install Visual Studio "old" SDK

Thank you!

**Reply by migueesc123** [reply]
let me give this a try and report back. I remember I tried it and it worked for me

**Reply by gtherezan** [reply]
Hi Miguel!

Sorry to bother, but do you have an update?

**Comment by migueesc123** [comment]
Just updated the repo. It should work now.

Please give it a try and let me know  if you have any issues

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by gtherezan** [reply]
Now it works fine!

Thank you very much!!
